servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcClientBuilder.java

```diff
@@ -93,7 +127,11 @@
      * @param strategy {@link GrpcExecutionStrategy} to use.
      * @return {@code this}.
      * @see GrpcExecutionStrategies
+     * @deprecated Call {@link #initializeHttp(HttpInitializer)} and use
+     * {@link SingleAddressHttpClientBuilder#executionStrategy(HttpExecutionStrategy)} on the {@code builder} instance
+     * by implementing {@link HttpInitializer#initialize(SingleAddressHttpClientBuilder)} functional interface.
      */
+    @Deprecated
```

Comment:
If we deprecate this method, we should also deprecate `GrpcExecutionStrategy` and `GrpcExecutionStrategies`.